src/Microsoft.ApplicationInsights/DataContracts/TelemetryContext.cs

```diff
@@ -132,13 +132,13 @@ public LocationContext Location
             get
             {
                 var result = new Dictionary<string, string>();
-                this.component?.UpdateTags(result);
-                this.device?.UpdateTags(result);
-                this.cloud?.UpdateTags(result);
-                this.session?.UpdateTags(result);
-                this.user?.UpdateTags(result);
-                this.operation?.UpdateTags(result);
-                this.location?.UpdateTags(result);
+                this.Component.UpdateTags(result);
```

Comment:
Changing this method defeats the purpose of making the backing fields lazy. I don't think this change is necessary.

Reply:
at a casual glance, it looked like an identical bug that hadn't been discovered yet. After investigating I agree. This method is used as an export, and the null properties should remain null. I will revert this change.

@pharring what do you think about `this.Internal.UpdateTags(result)`? Should that remain as is?

Reply:
@MS-TimothyMothra yes. Internal is different because it is not backed by a lazy-initialized field. It should remain as-is.